src/components/post-form/PostForm.js

```diff
@@ -126,6 +134,13 @@ class PostForm extends Component {
           this.renderUsersSelect()
         }
         {/* todo 2: добавить кнопку, по нажатию на которую будет вызываться метод onReset*/}
+
+        {/*не знаю як зробити так, щоб button не створював нового пустого поста*/}
+        <div className='may-reset-forms'>
+          <button onClick={this.onReset}>Reset form</button>
+          {/*тому просто  дівку підстилізував під button і все ок*/}
+              <div onClick={this.onReset} className='may-reset-forms-custom'>Reset</div>
```

Comment:
This variant is good

if you want to use simple button you can use button with `type="button"`